src/pages/CommitQueue.tsx

```diff
@@ -44,26 +45,30 @@ export const CommitQueue = () => {
         title="Commit Queue"
         badge={
           <Badge variant="darkgray">
-            {queue.length} Item {queue.length > 1 && "s"}
+            {queue ? queue.length : 0} Item
```

Comment:
I think "0 items" is actually the correct way to say it

Reply:
i also had trouble reading this